talk/owt/sdk/p2p/p2ppeerconnectionchannel.cc

```diff
@@ -1296,7 +1296,9 @@ void P2PPeerConnectionChannel::DrainPendingMessages() {
 }
 void P2PPeerConnectionChannel::DrainPendingRemoteCandidates() {
   rtc::scoped_refptr<webrtc::PeerConnectionInterface> temp_pc_ = GetPeerConnectionRef();
-  {
+  // If we didn't get a connection ref, that means that the connection already got closed
+  // and pending remote candidates have already been cleared.
+  if (temp_pc_) {
```

Comment:
This can go in line 1303 as:
`if (temp_pc && temp_pc_->remote_description()) {`

and the `rtc::CritScope cs(&pending_remote_candidates_crit_);` moved to inside the if block at line 1304.

Reply:
@dtag this seems to be unused. Did you mean to `GetPeerConnectionRef()` and do something with it?
https://github.com/AmbientAI/owt-client-native/blob/50c201dfd05627808504a52750aad0abaca532e2/talk/owt/sdk/p2p/p2ppeerconnectionchannel.cc#L549